distribution/docker/Dockerfile

```diff
@@ -47,12 +47,13 @@ RUN ["/busybox/busybox", "--install", "/bin"]
 
 RUN addgroup -S -g 1000 druid \
  && adduser -S -u 1000 -D -H -h /opt/druid -s /bin/sh -g '' -G druid druid \
- && mkdir -p /opt/druid/var \
- && chown -R druid:druid /opt \
- && chmod 775 /opt/druid/var
+ && mkdir -p /opt
```

Comment:
I don't think creating `/opt` is necessary, the COPY will take care of it

Reply:
fixed